<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherDownloaderDlg</class>
 <widget class="QDialog" name="WeatherDownloaderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Downloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="StatescomboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Station</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="SelectStationcomboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateTimeEdit" name="dateEditStart"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateTimeEdit" name="dateEditEnd"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="FetchDataBtm">
         <property name="text">
          <string>Retrive Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExporttoCSV">
         <property name="text">
          <string>Export to CSV</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="MetaDatatextBrowser"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Meta Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="textBrowserPrecip"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
